OCAExplorer/src/components/Demo.tsx

```diff
@@ -130,6 +137,6 @@ export function Demo({runDemo, theme, resetFunc }: {runDemo: DemoState, theme: T
                zIndex: 1000,
              }
 	   }}
-	   run={runDemo != "NotRunning"}
+	   run={runDemo != DemoState.NotRunning}
```

Comment:
The logic here should be to run t he demo when the demo state is not `SeenDemo`, everything else should trigger the demo.

Reply:
Wouldn't this end up being `NotSeenDemo` or `SeenDemo`?